src/registry-client/src/index.ts

```diff
@@ -133,12 +164,14 @@ const { version } = loadPackageJson(
 ) as {
   version: string
 }
+/* c8 ignore start - we do test this, but coverage fails */
```

Comment:
This is probably because https://github.com/vltpkg/vltpkg/blob/main/CONTRIBUTING.md#test-coverage-is-failing-but-it-shouldnt-be.

Something with using type stripping messes with coverage but only when doing `import from X` and `t.mockImport(X)` in the same test.